models/ape_store/base/ape_store_base_schema.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+
+models:
+  - name: ape_store_base_trades
```

Comment:
Do you want to add this `ape_store_base_trades` into any roll-up spell?

Reply:
no, it should be a standalone spell. once they add more chains i'll add a rollup @Hosuke

Reply:
In this case do you want to add a `post_hook` to the spell?

Reply:
yes! will do that

Reply:
done! @Hosuke

Reply:
can we actually remove the post hook to expose on data explorer? if we know this isn't the end spell, even if final not built yet, i'd rather hold off. we need to clean out the data explorer rather than adding more at the moment. when in doubt, we should be looking to add if it's a final level spell only and is built for the overall community (i.e. a `dex.trades` or `dex.bot_trades` type)